datapipelines/src/test/java/io/customer/datapipelines/testutils/core/UnitTestDelegate.kt

```diff
@@ -25,7 +25,7 @@ import io.mockk.spyk
  * by child classes which allows running tests with or without depending on Android
  * context and resources.
  */
-abstract class UnitTest {
+abstract class UnitTestDelegate {
```

Comment:
This is renamed to delegate because we cannot inherit multiple classes. So future updates to base test classes in `datapipelines` will utilize it through composition to avoid code duplication and simplify base setup.